test/src/main/java/org/apache/accumulo/test/TransportCachingIT.java

```diff
@@ -88,7 +90,14 @@ public void testCachedTransport() throws InterruptedException {
           // Get a transport (cached or not)
           first = pool.getAnyTransport(servers, true).getSecond();
         } catch (TTransportException e) {
-          log.warn("Failed to obtain transport to {}", servers);
+          failTransportCount++;
+          if (failTransportCount > MAX_FAILURES) {
+            log.warn(
+                "Failed to obtain transport to {} over 1000 times and will stop outputing this log message",
+                servers);
+            break;
```

Comment:
I am not sure what the desired behavior should be but it looks like this `break` will break out of the while loop. Therefore, the log message is not really correct. It's not that you are no longer printing messages, but rather no longer trying to obtain transport. Is that the desired behavior?